crates/bevy_window/src/window.rs

```diff
@@ -794,7 +819,9 @@ pub enum CompositeAlphaMode {
     Inherit = 4,
 }
 
-/// Defines the way a window is displayed
+/// Defines the way a [`Window`] is displayed, potentially using a given size.
+///
+/// The "given size" is in the [`Window`]'s [`resolution`](WindowResolution).
```

Comment:
There are references to the "given size" in the doc of each variant of the enum, though?

Reply:
Ah, I was hoping we could reword those to match. Because they don't store values I tend to find it quite confusing that they talk about "given size".

I also think that the more direct phrasing of the struct docs here is much clearer.

Reply:
Proposed change: change "given size" to "window resolution size" in the variants

Also precise "physical size"
```rust
/// Defines the way a [`Window`] is displayed,
/// potentially using the physical size in the [`Window`]'s [`resolution`](WindowResolution).
#[derive(Default, Debug, Clone, Copy, PartialEq, Eq, Reflect, FromReflect)]
#[cfg_attr(
    feature = "serialize",
    derive(serde::Serialize, serde::Deserialize),
    reflect(Serialize, Deserialize)
)]
#[reflect(Debug, PartialEq)]
pub enum WindowMode {
    /// The window should take a portion of the screen, using the window resolution size.
    #[default]
    Windowed,
    /// The window should be borderless and use the full size of the screen.
    BorderlessFullscreen,
    /// The window should be borderless, take the full size of the screen, and render at desktop resolution.
    /// The app will use the closest supported size from the window resolution size and scale it to fit the screen.
    SizedFullscreen,
    /// The window should take the full size of the screen by using the maximum supported size.
    Fullscreen,
}
```

Reply:
Good I like that!

Reply:
Resolved by https://github.com/bevyengine/bevy/pull/8858/commits/f40d539ebb9fb336a87bd91dc4b1f591e7826c10